wgpu-native/Cargo.toml

```diff
@@ -28,16 +28,16 @@ bitflags = "1.0"
 copyless = "0.1"
 lazy_static = "1.1.0"
 log = "0.4"
-hal = { package = "gfx-hal", version = "0.2.1" }
-gfx-backend-empty = { version = "0.2.1" }
-gfx-backend-vulkan = { version = "0.2", optional = true }
-gfx-backend-dx11 = { version = "0.2.1", optional = true }
-gfx-backend-dx12 = { version = "0.2.1", optional = true }
-gfx-backend-metal = { version = "0.2.2", optional = true }
-gfx-backend-gl = { version = "0.2", optional = true }
+hal = { package = "gfx-hal", version = "0.3.0" }
+gfx-backend-empty = { version = "0.3.0", features = ["winit"] }
```

Comment:
`winit` feature isn't the default, we only need it enabled when something like `window-winit` feature is enabled here

Reply:
I tried doing: ```window-winit = ["winit", "gfx-backend-empty/winit", "gfx-backend-vulkan/winit", "gfx-backend-dx11/winit", "gfx-backend-dx12/winit", "gfx-backend-metal/winit", "gfx-backend-gl/winit", "gfx-backend-gl/glutin"]```
But that doesn't work as it now tries to compile every backend.

Reply:
Unless you can explain how this is supposed to work, I'll leave it for someone else to investigate. I have not regressed any functionality here, the optional winit feature is new gfx hal 0.3